crates/graphql/src/schema/objects/nft.rs

```diff
@@ -86,6 +89,43 @@ pub struct NftOwner {
     pub address: String,
 }
 
+#[derive(Debug, Clone, GraphQLObject)]
+pub struct NftActivity {
+    pub address: String,
+    pub metadata: String,
+    pub auction_house: String,
+    pub price: scalars::Lamports,
+    pub created_at: DateTime<Utc>,
+    pub wallets: Vec<String>,
+    pub activity_type: String,
+}
+
+impl TryFrom<models::NftActivity> for NftActivity {
+    type Error = std::num::TryFromIntError;
+
+    fn try_from(
+        models::NftActivity {
+            address,
+            metadata,
+            auction_house,
+            price,
+            created_at,
+            wallets,
+            activity_type,
+        }: models::NftActivity,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            address,
+            metadata,
+            auction_house,
```

Comment:
Likely want to conver this to the auction house object so holaplex.com can show the marketplace associated to the activity.